### DISCUSSION TITLE
ICI - inter core interconnects - in Single TPU vs Slices/PODs

### DISCUSSION THREAD

**Discussion by cpcpython** [jax-ml/jax#24713] [General] [answered]
Hi List

I was reading web docs on TPUs and a Paper by Jouppi et al: (A Domain Specific Supercomputer for Training Deep Neural Networks) Pls see: [https://dl.acm.org/doi/pdf/10.1145/3360307]for the paper and See Fig.2 , there for a plot of ICI ( Inter-Core Interconnect ).

I have 2 simple/general doubts (sorry for the basic questions):

1. If I use a single v2-8 TPU does these 8 cores are connected with ICI ?
2. Or these ICIs are only valid IF I use multi-host pod/slices ?

I hope some TPU experts can answer my basic queries... Thanks in advance

Krishna

**Comment by hawkinsp** [comment, marked as answer]
Yes, a v2-8 has all of its chips connected via ICI in a ring.

**Reply by cpcpython** [reply]
Thanx Hawkins!
